core/src/test/java/io/substrait/extendedexpression/ExtendedExpressionRoundTripTest.java

```diff
@@ -0,0 +1,229 @@
+package io.substrait.extendedexpression;
```

Comment:
optional nit: the tests might be more readable with helper functions for `createTestExpressionReference()`, `createTestAggregateReference()` and `verifyRoundTrip()`. It could help reduce code duplication, too.